frontend/src/router/index.js

```diff
@@ -224,9 +239,12 @@ const router = new Router({
 
 router.beforeEach((to, from, next) => {
 
+  //If path name is invalid, redirect to 404 page.
+  if (!validPathNames.includes(to.name))
```

Comment:
This array (validPathNames) must be updated with each new path, at the risk of the user being redirected to the 404 page on an existing URL that isn't included in the array.

Reply:
This can be solved by using `to.name`. This will be null if an unkown URL is passed, therefore we can redirect to a 404 page on a null `to.name` value. You'll be able to see this in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/42/commits/8bde178d83d101e9a7e935c820564045534245a5)